man/systemd.exec.xml

```diff
@@ -3690,6 +3690,15 @@ StandardInputData=V2XigLJyZSBubyBzdHJhbmdlcnMgdG8gbG92ZQpZb3Uga25vdyB0aGUgcnVsZX
           system.</para></listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><varname>$TRIGGER_REASON</varname></term>
+
+          <listitem><para>If the unit was activated dynamically (e.g.: a corresponding path unit), the reason
+          for the activation will be listed in this variable. Note that this is provided in a best-effort way.
+          For example, there might be multiple triggers firing at any given time, and only one will be reported,
```

Comment:
I think it also matters to say that all kinds of triggering events can be queued and colaesced in the queue. So it's not "firing at any given time", it's also "coalesced if happening one after the other". if you follow what I mean